src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Shared/LoginDisplay.IndividualMsalAuth.razor

```diff
@@ -1,23 +1,20 @@
 ﻿@using Microsoft.AspNetCore.Components.Authorization
 @using Microsoft.AspNetCore.Components.WebAssembly.Authentication
-
 @inject NavigationManager Navigation
-@inject SignOutSessionStateManager SignOutManager
 
 <AuthorizeView>
     <Authorized>
         Hello, @context.User.Identity?.Name!
-        <button class="nav-link btn btn-link" @onclick="BeginLogout">Log out</button>
+        <button class="nav-link btn btn-link" @onclick="BeginSignOut">Log out</button>
```

Comment:
Is there any coherent rule about when we say "sign out" vs "log out"?

It's pretty weird to have a link labelled "log out" but the method is called "sign out", which then redirects to a URL containing "logout".

What stops us from using "log out" consistently in this code?

Reply:
```suggestion
        <button class="nav-link btn btn-link" @onclick="BeginLogOut">Log out</button>
```

Reply:
I have changed it, we had a discussion about this like 3 years ago, but I don't remember the details.

Part of what we wanted to make sure was that these things weren't confused with identity concepts, and that's what motivated some of the naming. I have changed it to be more consistent here.